src/x3f_extract.c

```diff
@@ -63,12 +63,14 @@ static void usage(char *progname)
           "   -histogram      Dump histogram as csv file\n"
           "   -loghist        Dump histogram as csv file, with log exposure\n"
 	  "APPROPRIATE COMBINATIONS OF MODIFIER SWITCHES\n"
-	  "   -color <COLOR>  Convert to RGB color\n"
-	  "                   (sRGB, AdobeRGB, ProPhotoRGB)\n"
+	  "   -color <COLOR>  Convert to RGB color space\n"
+	  "                   (none, sRGB, AdobeRGB, ProPhotoRGB)\n"
+	  "                   none means neither scaled, gamma nor colorspace\n"
+	  "                   color do not affect DNG\n"
           "   -unprocessed    Dump RAW without any preprocessing\n"
```

Comment:
none means neither scaling, gamma nor color space
This switch does not affect DNG output